.github/workflows/tests.yml

```diff
@@ -18,11 +18,11 @@ jobs:
       - name: Install PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 7.4
+          php-version: 8.0
```

Comment:
Does this mean we are not compatible with PHP 7 anymore? Or we just don't test 7?

Reply:
It's only to run the linter, it does not impact our compatibility.